bundles/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/dpt/KNXCoreTypeMapper.java

```diff
@@ -571,6 +583,154 @@ public KNXCoreTypeMapper() {
         defaultDptMap.put(HSBType.class, DPTXlatorRGB.DPT_RGB.getID());
     }
 
+    /*
+     * This function computes the target unit for type conversion from OH quantity type to DPT types.
+     * Calimero library provides units which can be used for most of the DPTs. There are some deviations
+     * from the OH unit scheme which are handled.
+     */
+    private String quantityTypeToDPTValue(QuantityType<?> qt, int mainNumber, int subNumber, String targetOhUnit)
```

Comment:
If I correctly understand how the method is called, the last parameter is in fact a DP unit.
I would suggest to name the parameter `dpUnit` and then create a local variable named targetOhUnit that you will initialize with `dpUnit`

Reply:
done